setup.py

```diff
@@ -419,11 +421,67 @@ def get_tag(self):
     cmdclass['build_src'] = DummyBuildSrc
     cmdclass['build_ext'] = CheckingBuildExt
 
+# ----------------------------------------------------------------------
+# Preparation of compiler arguments
+
 if sys.byteorder == 'big':
     endian_macro = [('__BIG_ENDIAN__', '1')]
 else:
     endian_macro = [('__LITTLE_ENDIAN__', '1')]
 
+
+if is_platform_windows():
+    extra_compile_args = []
+else:
+    # args to ignore warnings
+    extra_compile_args = ['-Wno-unused-function']
+
+
+# enable coverage by building cython files by setting the environment variable
+# "linetrace" (with a Truthy value)
+linetrace = os.environ.get('linetrace', False)
```

Comment:
yeah this need to be uppercase and should be like PANDAS_CYTHON_COVERAGE or somesuch